libraries/app/application.cpp

```diff
@@ -997,6 +1016,7 @@ void application::set_program_options(boost::program_options::options_descriptio
    configuration_file_options.add_options()
          ("p2p-endpoint", bpo::value<string>(), "Endpoint for P2P node to listen on")
          ("p2p-max-connections", bpo::value<uint32_t>(), "Maxmimum number of incoming connections on P2P endpoint")
+         ("p2p-parameters", bpo::value<string>()->default_value("{}"), "P2P network parameters")
```

Comment:
The default value needs to mean something. The only documentation of what should go here, is the source code itself. This needs to be the default allocation `node_configuration`.